docs/reference/schema.md

```diff
@@ -0,0 +1,539 @@
+# Schema reference
+
+The schema provides the authoritative definition of the structure of Risk Data Library Standard (RDLS) data, the meaning of each field, and the rules that must be followed to publish RDLS data. It is used to validate the structure and format of RDLS data.
+
+For this version of RDLS, the canonical URL of the schema is \[\]\](). Use the canonical URL to make sure that your software, documentation or other resources refer to the specific version of the schema with which they were tested.
+
+This page presents the schema in tables with additional information in paragraphs. You can also [view the schema in an interactive browser](browser.md) or [download it as JSON Schema](../../docs/_readthedocs/html/rdl_schema_0.1.json).
+
+```{note}
+   If any conflicts are found between the text on this page and the text within the schema, the text within the schema takes precedence.
+```
+
+The RDLS schema covers [dataset attributes](#dataset), [resource attributes](#resource) and four risk-specific components:
+
+- [Hazard](#hazard): main hazard type, specific process, trigger of the hazard, occurrence frequency of event, intensity unit to measure the process and analytical method.
+- [Exposure](#exposure): asset category, occupancy and specific taxonomy, cost type and value.
```

Comment:
```suggestion
- [Exposure](#exposure): asset category, specific taxonomy, cost type and value.
```

It was agreed to remove occupancy from the schema as it can vary within a resource